src/onLCP.ts

```diff
@@ -96,17 +96,25 @@ export const onLCP = (
         report(true);
       });
 
+      // Need a separate wrapper to ensure the `runOnce` function above is
+      // common for all three functions
+      const stopListeningWrapper = (e: Event) => {
```

Comment:
Nit: we've used `event` (rather than `e`) in other places in this library so I'd rename to be consistent.